doc/workshops/spyder_workshop_2.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Scientific Computing and Visualization with Spyder
+
+Created on Thu May 20 10:17:27 2021
+
+@author: Spyder Team
+"""
+
+# %% Import the libraries
+import scipy.stats as stats
+import pandas as pd
+
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+# %% Load raw data (csv) and save it in parquet format:
+data = pd.read_csv("parsed_data_public.csv", low_memory=False)
+data.to_parquet("parsed_data_public.parquet")
+
+
+# %% Load raw data (parquet)
+data = pd.read_parquet("parsed_data_public.parquet")
```

Comment:
Why are you loading the data first first as csv, then writing it to parquet, then loading it again? If we're distributing the data for this workshop as a parquet file, we should only load the parquet file, right?